.env.example

```
@@ -62,8 +62,9 @@ NUXT_OPENAI_API_KEY=your-openai-api-key-here
 # For testing
 NUXT_TEST_DATABASE_URL=postgres://postgres:@localhost:5432/getquillio_test
 NUXT_TEST_APP_URL=http://localhost:3000 # App URL for e2e test
-NUXT_TEST_EMAIL=test@getquillio.com
+NUXT_TEST_EMAIL=test@getquillio.com # Must exist in the test DB and be verified
 NUXT_TEST_PASSWORD=test-password
+NUXT_TEST_ORGANIZATION_ID= # Optional: force active org for e2e tests
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)